### PR TITLE
Create how we use PRs page

### PR DESCRIPTION
**Pull request by U8NWXD**

### CODE REVIEW

File: How-we-use-pull-requests.md
Comment (DubeySandeep) [inline How-we-use-pull-requests.md:33]: The PR is expected to be reviewed by the main/primary reviewer first (Currently decided by OppiaBot) and once they approve, all other code owners get assigned!

You can find more details [here](https://docs.google.com/document/d/1y2_3Ns0V00QESZy3gswc413Rki-m1exTNDt3uVcu7w0/edit#).

File: How-we-use-pull-requests.md
Comment (DubeySandeep) [inline How-we-use-pull-requests.md:44]: Let's not talk about issues here*

File: How-we-use-pull-requests.md
Comment (DubeySandeep) [inline How-we-use-pull-requests.md:3]: Not sure about the title of this page, this dock doesn't talk about the uses of the PRs, right?

File: How-we-use-pull-requests.md
Comment (DubeySandeep) [inline How-we-use-pull-requests.md:40]: Can you please check with the OppiaBot team on what are the bot's expectations from draft PRs? (I'm unable recall all the expectations!)

File: How-we-use-pull-requests.md
Comment (DubeySandeep) [inline How-we-use-pull-requests.md:68]: I don't see anything related to "Juggling" here, are we expecting to explain "how-tos" here or just inform that contributors can create multiple PRs in parallel? (If the latter one then maybe change the heading?)

File: How-we-use-pull-requests.md
Comment (DubeySandeep) [inline How-we-use-pull-requests.md:8]: Should we call it a Pull request lifecycle? (Considering all other headings have "pull request")

**Review by DubeySandeep — CHANGES_REQUESTED** [review]
@U8NWXD I've left a few comments, PTAL!

File: How-we-use-pull-requests.md
Comment (U8NWXD) [reply How-we-use-pull-requests.md:33]: Isn't that what's written? Specifically these sentences:

> Oppiabot will assign one developer (typically the developer from the changelog label) to take a first pass. Oppiabot will assign the other code owners to review once the first pass is complete.

File: How-we-use-pull-requests.md
Comment (U8NWXD) [reply How-we-use-pull-requests.md:40]: Based on the recent draft PRs I've opened, I'm pretty sure that Oppiabot just expects to see `[skip ci]` or `[ci skip]` in the commit message.

@vojtechjelinek is this correct?

File: How-we-use-pull-requests.md
Comment (U8NWXD) [reply How-we-use-pull-requests.md:44]: done

File: How-we-use-pull-requests.md
Comment (U8NWXD) [reply How-we-use-pull-requests.md]: I've renamed it to "Pull requests at Oppia"

File: How-we-use-pull-requests.md
Comment (U8NWXD) [reply How-we-use-pull-requests.md]: This is really just about handling multiple PRs in parallel. I've updated the heading

File: How-we-use-pull-requests.md
Comment (U8NWXD) [reply How-we-use-pull-requests.md]: done

**Comment by U8NWXD** [comment]
@DubeySandeep I've addressed your comments PTAL

File: How-we-use-pull-requests.md
Comment (U8NWXD) [reply How-we-use-pull-requests.md]: @vojtechjelinek has confirmed offline that Oppiabot just expects to see `[skip ci]` or `[ci skip]`

File: How-we-use-pull-requests.md
Comment (DubeySandeep) [reply How-we-use-pull-requests.md]: > reviews will automatically be requested from all code owners

Ah, I misinterpreted "requested" as assigned! (Sorry!)

> You can find more details here.

Just to confirm, this wiki page follows the process defined in the doc linked above, right?

File: Pull-requests-at-Oppia.md
Comment (DubeySandeep) [inline Pull-requests-at-Oppia.md:6]: Do we need to change the id name here to match the updated title?

**Review by DubeySandeep — APPROVED** [review]
LGTM, @seanlip Can you please take a final pass?

File: Pull-requests-at-Oppia.md
Comment (DubeySandeep) [reply Pull-requests-at-Oppia.md:6]: Also, the heading names are not updated here?

File: Pull-requests-at-Oppia.md
Comment (seanlip) [inline Pull-requests-at-Oppia.md:32]: Perhaps elaborate more on what "Reviews at this stage are at the discretion of the author" means. In particular what is the author supposed to do / what decision are they supposed to make here, and how do they communicate this decision?

File: Pull-requests-at-Oppia.md
Comment (seanlip) [inline Pull-requests-at-Oppia.md:40]: Here and elsewhere, CircleCI is one word.

File: Pull-requests-at-Oppia.md
Comment (seanlip) [inline Pull-requests-at-Oppia.md:42]: Why do we still refer to Travis?

File: Pull-requests-at-Oppia.md
Comment (seanlip) [inline Pull-requests-at-Oppia.md:64]: The "Git allows you to do this seamlessly!" part seems a bit of a throwaway. Either elaborate on it or link it to a separate page -- it's a reasonable "new contributor confusion" that they aren't sure how to make a parallel PR and we should try to explain it somehow.

The next point is a separate point and should be given its own prominence (perhaps as a separate paragraph).

File: Pull-requests-at-Oppia.md
Comment (seanlip) [inline Pull-requests-at-Oppia.md:44]: This list is kind of overwhelming. Some of this stuff is automatically added and some is meant to be added by the reviewer.

Perhaps explain:
- this is the stuff you should add (then have a bullet list saying "if your PR has X, add this label"); 
- this is the stuff that may get automatically added and what it means (this bullet list can be based on label);
- here's what you do if you don't have permissions to add labels.

**Review by seanlip — CHANGES_REQUESTED** [review]

File: Pull-requests-at-Oppia.md
Comment (U8NWXD) [reply Pull-requests-at-Oppia.md:44]: done

File: Pull-requests-at-Oppia.md
Comment (U8NWXD) [reply Pull-requests-at-Oppia.md]: done

File: Pull-requests-at-Oppia.md
Comment (U8NWXD) [reply Pull-requests-at-Oppia.md]: I removed Travis and added GitHub Actions

File: Pull-requests-at-Oppia.md
Comment (U8NWXD) [reply Pull-requests-at-Oppia.md]: done

File: Pull-requests-at-Oppia.md
Comment (U8NWXD) [reply Pull-requests-at-Oppia.md]: done

File: Pull-requests-at-Oppia.md
Comment (U8NWXD) [reply Pull-requests-at-Oppia.md]: fixed

**Comment by U8NWXD** [comment]
@seanlip PTAL

File: Pull-requests-at-Oppia.md
Comment (seanlip) [inline Pull-requests-at-Oppia.md:50]: updates --> update

**Review by seanlip — CHANGES_REQUESTED** [review]
Thanks! This should be my last round of comments.

File: Pull-requests-at-Oppia.md
Comment (U8NWXD) [reply Pull-requests-at-Oppia.md]: done

File: Pull-requests-at-Oppia.md
Comment (U8NWXD) [reply Pull-requests-at-Oppia.md]: done

File: Pull-requests-at-Oppia.md
Comment (U8NWXD) [reply Pull-requests-at-Oppia.md]: done

File: Pull-requests-at-Oppia.md
Comment (U8NWXD) [reply Pull-requests-at-Oppia.md]: done

**Comment by U8NWXD** [comment]
@seanlip PTAL

**Review by seanlip — APPROVED** [review]
Thanks -- LGTM

**Pull request merged**
